README.rst

```diff
@@ -1,76 +1,51 @@
+This is a micro-frontend application responsible for displaying and maintaining tools for skills development. This is the home for the B2C Skills Builder. 
 
-After Copying The Template
---------------------------
+.. note::
+   This MFE is currently under development. Aperture is in the process of extracting the Skills Builder from the Profile MFE to this location. See the **Development Roadmap** items linked below.
 
-You'll want to do a find-and-replace to replace all instances of ``frontend-template-application`` with the name of your new repository.  Also edit index.html to replace "Application Template" with a friendly name for this application that users will see in their browser tab.
-
-**Prerequisite**
-
-`Devstack <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/installation/index.html>`_.  If you start Devstack with ``make dev.up.ecommerce`` that should give you everything you need as a companion to this frontend.
-
-**Installation and Startup**
-
-In the following steps, replace "frontend-template-application' with the name of the repo you created when copying this template above.
-
-1. Clone your new repo:
-
-  ``git clone https://github.com/openedx/frontend-template-application.git``
-
-2. Use node v12.x.
-
-   The micro-frontend build scripts support node 12.  Using other major versions of node *may* work, but is unsupported.  For convenience, this repository includes an .nvmrc file to help in setting the correct node version via `nvm <https://github.com/nvm-sh/nvm>`_.
-
-3. Install npm dependencies:
-
-  ``cd frontend-template-application && npm install``
-
-4. Update the application port to use for local development:
-
-   Default port is 8080. If this does not work for you, update the line `PORT=8080` to your port in all .env.* files
+Development
+------------
 
-5. Start the dev server:
+Start the development server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In this project, install requirements and start the development server by running:
 
-  ``npm start``
+.. code-block::
 
-The dev server is running at `http://localhost:8080 <http://localhost:8080>`_ or whatever port you setup.
+   cd frontend-app-skills
+   npm install
+   npm start # The server will run on port 8080
 
-Making Your New Project's README File
--------------------------------------
+Once the dev server is up visit, http://localhost:8080
 
-Move the file ``README-template-frontend-app.rst`` to your project's ``README.rst`` file. Please fill out all
-the sections - this helps out all developers understand your MFE, how to install it, and how to use it.
+Environment Variables
+---------------------------------
 
-Project Structure
------------------
+In order to run the Skills Builder, you'll need to add Algolia keys to the ``.env.development`` file. If you have access to the `edx-internal repo`_, you can copy the ones for the Skills MFE's stage configuration. Alternatively, reach out to **@openedx/2u-aperture**.
```

Comment:
Another place where we want to update the team name.

Reply:
lol did this already get updated once? It should be the edx GH team